bitnami/external-dns/templates/deployment.yaml

```
@@ -37,7 +37,7 @@ spec:
       {{- if .Values.rbac.create }}
```

Thanks for your changes. I think we also need to check both parameters in this condition, what do you think?
```suggestion
      {{- if and .Values.rbac.create .Values.rbac.serviceAccount.create }}
```